pages/mesosphere/dcos/services/elastic/3.2.0-7.9.3/how-to-guides/exposing-kibana/index.md

```diff
@@ -0,0 +1,225 @@
+    <!-- [Edge-LB installation instructions](/mesosphere/dcos/services/edge-lb/getting-started/installing/). -->
+
+
+## Create an Edge-LB pool for Kibana
+
+The following command will create an Edge-LB pool task running on one of your DC/OS cluster's public agents, which will allow Kibana to be accessed from outside the cluster network, given that the selected port on the agent machine is open.
+
+In this example, we'll expose a Kibana service named `/production/kibana` through HTTP, on port `80`. It will be accessible on `http://$public_agent_ip_or_url:80`.
+
+Note that in this example:
+- the Edge-LB pool that will be created is named `kibana`
+- its backend name is `kibana-backend`
+
+It is not a requirement that these match any configuration options related to the actual Kibana service, so you could name them differently.
+
+The pool fields that actually map to the actual Kibana service are under `haproxy.backends`:
+- `rewriteHttp.path.fromPath` should match the Kibana Marathon app service path
+- `services.endpoint.portName` should match the Kibana Marathon app port name
+- `services.marathon.serviceID` should match the Kibana service name
+
+1. Let's get the remaining configuration parameters that will map the Edge-LB pool to the actual Kibana service. We'll use them in the pool configuration. Make sure to use a different name or port based on your needs.
+
+    ```bash
+    kibana_service_name="/production/kibana"
+    kibana_proxy_port=80
+    kibana_service_path="/service/${kibana_service_name}"
+    kibana_port_name="$(dcos marathon app show "${kibana_service_name}" | jq -r '.portDefinitions[0].name')"
+    ```
+
+    ```bash
+    echo "{
+      \"apiVersion\": \"V2\",
+      \"role\": \"slave_public\",
+      \"name\": \"kibana\",
+      \"count\": 1,
+      \"haproxy\": {
+        \"stats\": {
+          \"bindPort\": 9090
+        },
+        \"frontends\": [
+          {
+            \"bindPort\": ${kibana_proxy_port},
+            \"linkBackend\": {
+              \"defaultBackend\": \"kibana-backend\"
+            },
+            \"protocol\": \"HTTP\"
+          }
+        ],
+        \"backends\": [
+          {
+            \"name\": \"kibana-backend\",
+            \"protocol\": \"HTTP\",
+            \"rewriteHttp\": {
+              \"path\": {
+                \"fromPath\": \"${kibana_service_path}\",
+                \"toPath\": \"/\"
+              }
+            },
+            \"services\": [
+              {
+                \"marathon\": {
+                  \"serviceID\": \"${kibana_service_name}\"
+                },
+                \"endpoint\": {
+                  \"portName\": \"${kibana_port_name}\"
+                }
+              }
+            ]
+          }
+        ]
+      }
+    }" > kibana_pool.json
+    ```
+
+    Which will end up looking like:
+
+    `kibana_pool.json`
```

Comment:
I think there needs to be an extra line break between code snippets? Even if it's a file name? But I could be wrong on this part.
```suggestion
    `kibana_pool.json`
    
```